tests/antithesis/docker-compose.yml

```diff
@@ -1,5 +1,18 @@
 ---
 services:
+  # This is needed for creating non-root data folders on host.
+  # By default, if the folders don't exist when mounting, compose creates them with root as owner.
+  # With root owner, accessing the WAL files from local tests will fail due to an unauthorized access error.
+  init:
+    image: 'busybox:latest'
+    user: root
+    group_add:
+      - '${GROUP_ID:-root}'
+    volumes:
+      - ${ETCD_ROBUSTNESS_DATA_PATH:-/tmp/etcddata}0:/var/etcddata0
+      - ${ETCD_ROBUSTNESS_DATA_PATH:-/tmp/etcddata}1:/var/etcddata1
+      - ${ETCD_ROBUSTNESS_DATA_PATH:-/tmp/etcddata}2:/var/etcddata2
+    command: chown -R ${USER_ID:-root}:${GROUP_ID:-root} /var/etcddata0 /var/etcddata1 /var/etcddata2
```

Comment:
Do we need a fallback to `root`? Having defaults would allow users to call docker-compose directly, but I don't know if there is any value of it.